tests/integration/ambient/waypoint_test.go

```diff
@@ -788,6 +788,112 @@ spec:
 	})
 }
 
+func TestTCPRoute(t *testing.T) {
```

Comment:
We may need to add a comment:
Running `TestTCPRoute` and `TestTLSRoute` needs to set `PILOT_ENABLE_ALPHA_GATEWAY_API` to `true` since that are still under alpha Gateway API.